src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -293,12 +293,18 @@ public static class TextCatalog
             => new CustomStopWordsRemovingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(), outputColumnName, inputColumnName, stopwords);
 
         /// <summary>
-        /// Produces a bag of counts of ngrams (sequences of consecutive words) in <paramref name="inputColumnName"/>
-        /// and outputs bag of word vector as <paramref name="outputColumnName"/>
+        /// Create a <see cref="WordBagEstimator"/>, which takes the data from the column specified in <paramref name="inputColumnName"/>
+        /// to a new column: <paramref name="outputColumnName"/> and produces a vector of counts of n-grams.
```

Comment:
Would suggest a slightly different wording that uses 'maps' instead of 'takes':

which maps the column specified in <paramref name="inputColumnName"/> to a vector of counts of n-grams in a new column named <paramref name="outputColumnName"/>.